server.js

```diff
@@ -22,7 +23,38 @@ const app = express()
 app.use(cors())
 app.use(bodyParser.json())
 
-// Start defining your routes here
+// Start defining your RESTful end points/routes here
+app.get('/catalogue', (req, res) => {
+  res.json(netflixData)
+})
+
+app.get('/type/:type', (req, res) => {
+  let type = req.params.type.toLowerCase().replace(/[-_ ]/, "")
+  if (type === "tvshow" || type === "show" || type === "tv") {
```

Comment:
Nice! Just a little tip, you can make use of `Array.includes` to do a lot of checks like this, if you end up with a big list:

```
if (["tvshow", "show", "tv"].includes(type)) {
  ...
}
```